zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -9269,6 +9271,19 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/ProcessInstanceCreationStartInstruction"
+        runtimeInstructions:
+          description: |
+            Runtime instructions (alpha). List of instructions that affect the runtime behavior of
+            the process instance.
+            By default, the process instance will run until it terminates normally. If provided,
+            the process instance may finish earlier. Refer to specific instruction types for more
+            details.
```

🔧 Let's avoid mentioning that the instance may finish earlier. I have no clue what else we might use runtime instructions for in the future. Let's simplify refer to the specific type.

```suggestion
            Refer to specific instruction types for details.
```